tools/spatyper/spatyper.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="spatyper" name="Generate spa type identification" version="@VERSION@+galaxy0" profile="21.05">
+    <macros>
+        <token name="@VERSION@">0.3.3</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">spatyper</requirement>
+    </requirements>
+    <command detect_errors="aggressive"> <![CDATA[
+        spaTyper -f '${input1}' -r '${input2}' > '${output1}' 
+    ]]> </command>
+    <inputs>
+        <param type="data" name="input1" format="fasta" label="SPA Fasta file" help="Staphylococcus aureus - staphylococcal protein A fasta file. This will be assigned a type based on its repeat region."/>
+        <param type="data" name="input2" format="fasta" label="Repeats Library Multifasta" help="A multifasta representing a set of known repeat regions found in staphylococcal protein A. This can be downloaded at (http://spa.ridom.de/dynamic/sparepeats.fastas)"/>
```

Comment:
I think you have an extra `s` at the end here (as in `fasta` instead of `fastas`).

Reply:
good catch!